integration_test/robots/login_robot.dart

```diff
@@ -3,12 +3,21 @@ import 'package:flutter/material.dart';
 import 'package:tmail_ui_user/features/login/domain/model/recent_login_username.dart';
 import 'package:tmail_ui_user/features/login/presentation/login_view.dart';
 import 'package:tmail_ui_user/features/login/presentation/widgets/login_text_input_builder.dart';
+import 'package:tmail_ui_user/features/starting_page/presentation/twake_welcome/twake_welcome_view.dart';
+import 'package:tmail_ui_user/main/localizations/app_localizations.dart';
 
 import '../base/core_robot.dart';
 
 class LoginRobot extends CoreRobot {
   LoginRobot(super.$);
 
+  Future<void> expectWelcomeViewVisible() => ensureViewVisible($(TwakeWelcomeView));
+
+  Future<void> tapOnUseCompanyServer() async {
+    await $.pump(const Duration(seconds: 1));
```

Comment:
why do we need wait 1 seconds?

Reply:
The "Interactors" and their dependencies of "LoginBindings" will not load in time if we take the action immediately after `TwakeWelcomeView` is loaded

Reply:
So, we need to use another handle to make sure that the bindings are injected successfully. It's not a good idea to have random wait times like that. Should try more